docs/generated/settings/settings.md

```diff
@@ -0,0 +1,52 @@
+|                      SETTING                      |       TYPE        | DEFAULT  |                                                                   DESCRIPTION                                                                   |
+|---------------------------------------------------|-------------------|----------|-------------------------------------------------------------------------------------------------------------------------------------------------|
+| cloudstorage.gs.default.key                       | string            |          | if set, JSON key to use during Google Cloud Storage operations                                                                                  |
```

Comment:
Do you think we should put backticks around the settings, types, and defaults?

```
| `debug.panic_on_failed_assertions` | `boolean` | `false` | panic when an assertion fails rather than reporting |
```

Reply:
Done for name and value, type doesn't seem like a literal?